<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>451</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="add_cluster_button">
      <property name="text">
       <string>Add cluster</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="cluster_name_label">
          <property name="text">
           <string>Cluster name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="user_label">
          <property name="text">
           <string>User</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="host_label">
          <property name="text">
           <string>Host</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="port_label">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="auth_method_label">
          <property name="text">
           <string>Authentication Method</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="key_type_label">
          <property name="text">
           <string>Key type</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="cluster_name_edit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="user_edit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="host_edit"/>
        </item>
        <item>
         <widget class="QSpinBox" name="port_edit"/>
        </item>
        <item>
         <widget class="QComboBox" name="auth_method_box">
          <item>
           <property name="text">
            <string>PUBLIC_KEY</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ASK_EVERYTIME</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="key_type_box">
          <item>
           <property name="text">
            <string>RSA_KEY</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
